test/llilc_run.py

```diff
@@ -0,0 +1,105 @@
+﻿#!/usr/bin/env python
+#
+# title           : llilcrun.py
+# description     : Run a managed application using LLILC jit.
+#   
+
+import argparse
+import os
+import subprocess
+import sys
+
+llilcverbose = False
+
+def QuoteArg(arg):
+    '''Strip off any enclosing single quotes and enclose in double quotes'''
+    arg = '"' + arg.strip("'").strip('"') + '"'
+    return arg
+
+def UnquoteArg(arg):
+    ''' Remove single and double quotes from front and back of arg'''
+    return arg.strip("'").strip('"')
+
+def RunCommand(command):
+    ''' Run a command and return its exit code, optionally echoing it.'''
+    global llilcverbose
+    if llilcverbose:
+        sys.stderr.write (command + '\n')
+        sys.stdout.write (command + '\n')
+    error_level = subprocess.call(command, shell=True)
+    return error_level
+
+def main(argv):
+    '''
+    main method of script. arguments are script path and remaining arguments.
+    '''
+    global llilcverbose
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--llilc-dump-level', type=str, choices={'summary', 'verbose'}, 
```

Comment:
I did that because the unrecognized options are passed on to the program that is being run
and I did not want to accidentally process the options belonging to that 
program. That is also why I excluded short form options as they are very
likely to class with the program being run.